### ISSUE TITLE
Unknown 'speechSynthesis' error in listview-only page

### ISSUE THREAD

**Issue by martijnvankekem** [Samsung/TAU#368]
After opening the app, for a very specific group of people (I haven't found out which people do have this problem and which people don't. A friend of mine does have this problem while I have the same watch and don't have this problem), the app reports an error immediately on the following:

Uncaught TypeError: Cannot assign to read only property 'speechSynthesis' of object '#'
filename: file:///lib/tau/wearable/js/tau.js
line: 2195
column: 23

The page only contains a listview and there is nothing in my code calling anything close to a speechSynthesis property.

**Comment by lslachciak** [comment]
We have this on to do list. We need to consult web engine team for this issue too.
Maybe this issue is related somehow to accessibility support (speech synthesis) in web engine

**Comment by lslachciak** [comment]
@martijnvankekem could you attach sample app so we can try reproduce this issue?

**Comment by pkaczmarczy** [comment]
I assume that this problem is caused by fastMerge function. For tau 1.1.3 dist the line which throws an exception corresponds with this piece of code:
```
/**
 * Attach fields from second object to first object.
 * @method fastMerge
 * @param {Object} newObject
 * @param {Object} orgObject
 * @return {Object}
 * @static
 * @member ns.util.object
 */
fastMerge: function (newObject, orgObject) {
	var key;

	for (key in orgObject) {
		if (orgObject.hasOwnProperty(key)) {
			newObject[key] = orgObject[key];
		}
	}
	return newObject;
},
```

I have forced problem reproduction by calling ```tau.util.object.fastMerge(window, {speechSynthesis: {}});``` (this only reproduces error code, not the actual application behavior) and I get:
```
tau.js:2199 Uncaught TypeError: Cannot assign to read only property 'speechSynthesis' of object '#<Window>'
    at Object.fastMerge (tau.js:2199)
    at <anonymous>:1:17
```
We could add additional checks to merge functions (it would help, and may prevent further problems), but I still don't know why this method has been called.

**Comment by martijnvankekem** [comment]
I also don't know why this function would be called, and unfortunately, I can't reproduce where it came from, because I received this error message from users that are using my app. 

But, because this function returns the object, isn't this just a matter of returning a modified copy of 'newObject'?

So something like:

`modObject = {};
for (var attrname in newObject) { modObject[attrname] = newObject[attrname]; }
return modObject`

**Comment by pkaczmarczy** [comment]
This method modifies original object by design. (Such behavior is used in BaseWidget class, and this implementation expects that original object would be modified and conflicting properties would be overridden).

Modified version of method: https://github.com/Samsung/TAU/pull/379

**Comment by pkaczmarczy** [comment]
Dear @martijnvankekem ,
The PR has been merged, could you please let us know if it fixes the problem for you?

**Comment by lslachciak** [comment]
Fix will be part of 1.0.8 release
